packages/rocketchat-channel-settings-mail-messages/server/methods/mailMessages.js

```diff
@@ -55,7 +55,7 @@ Meteor.methods({
 		if (data.language !== 'en') {
 			const localeFn = Meteor.call('loadLocale', data.language);
 			if (localeFn) {
-				Function(localeFn)();
+				Function(moment.localeFn)();
```

Comment:
with the upcoming 0.57.2 I'd like to cherry pick this PR. @ggazzo could you give some feedback please. Thanks, Marcus